test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/DacFx/DacFxServiceTests.cs

```diff
@@ -843,15 +845,19 @@ private bool ValidateOptions(DeploymentOptions expected, DeploymentOptions actua
             foreach (var v in deploymentOptionsProperties)
             {
                 var defaultP = v.GetValue(expected);
+                var defaultPValue = defaultP != null ? defaultP.GetType().GetProperty("Value").GetValue(defaultP): defaultP;
                 var actualP = v.GetValue(actual);
+                var actualPValue = actualP.GetType().GetProperty("Value").GetValue(actualP);
 
                 if (v.Name == "ExcludeObjectTypes")
                 {
                     Assert.True((defaultP as ObjectType[])?.Length == (actualP as ObjectType[])?.Length, "Number of excluded objects is different not equal");
                 }
                 else
                 {
-                    Assert.True((defaultP == null && actualP == null) || (defaultP == null && (actualP as string) == string.Empty) || defaultP.Equals(actualP), $"Actual Property from Service is not equal to default property for {v.Name}, Actual value: {actualP} and Default value: {defaultP}");
+                    Assert.True((defaultP == null && actualP == null) || (defaultP == null && (actualP as string) == string.Empty) || defaultP.Equals(actualP)
```

Comment:
Can you break this up into multiple lines and/or multiple asserts and add comments?

Also, `String.IsNullOrEmpty(arg)` may be helpful in making this more readable.